app/templates/pdf/ticket_attendee.html

```diff
@@ -1,96 +1,212 @@
-            <span style="font-size: 20px;">{{ order.event.name }}</span><br><br>
-
-            {% if order.event.online %}
-                {% if order.event.location_name %}
-                    <span class="label">{{ _('Event taking place online and at %(event_location)s', event_location='') }}</span></br>
-                    <span style="font-size: 15px;"> {{order.event.location_name }}</span><br><br>
-                {% else %}
-                    <span class="label">{{ _('Online Event') }}</span><br><br>
-                {% endif %}
-            {% elif order.event.location_name %}
-                <span class="label">{{ _('At %(location_name)s', location_name='') }}</span></br>
-                <span style="font-size: 15px;"> {{order.event.location_name }}</span><br><br>
-            {% else %}
-                <span class="label">{{ _('Location or online event details to be announced') }}</span><br><br>
-            {% endif %}
-
-            <span class="label">{{ _('Date and Time') }}</span><br>
-            <span style="font-size: 15px;">{{ _('From: %(start_time)s', start_time='') }} {{ order.event.starts_at | datetime(order.event.timezone) }}</span><br>
-            <span style="font-size: 15px;">{{ _('To: %(end_time)s', end_time='') }} {{ order.event.ends_at | datetime(order.event.timezone) }}</span>
-            <br><br>
-
-            <span class="label">{{ _('Type') }}</span><br>
-            <span style="font-size: 20px;">{{ holder.ticket.name }}</span>
-            <br><br>
-
-            {% if order.event.after_order_message %}
-                <div id="notices">
-                    <div>{{ _('Organizer Message') }}:</div>
-                    <div class="notice">{% autoescape false %}{{ order.event.after_order_message | strip_tags | nl2br }}{% endautoescape %}</div>
+    <table id="header">
+        <tr>
+            <td>
+                <div id="billing_logo">
+                    {% if admin_info.admin_billing_logo %}
+                    <img src="{{admin_info.admin_billing_logo}}" alt="organization logo"/><br>
+                    {% endif %}
+                    <span id="app_name">{{app_name}}</span>
                 </div>
-                <br><br>
-            {% endif %}
-        </div>
+            </td>
+
+            <td align="right" style="vertical-align: bottom;"><span id="order_id">{{_('Order
+                    #%(invoice_number)s',invoice_number="")}}{{order.get_invoice_number()}}</span></td>
+        </tr>
+    </table>
+    <div class="main_content" id="main_content">
+        <table>
+            <tr>
+                <td id="event_name" colspan=2>
+                    <span>{{order.event.name}}</span><br>
+                    <span id="role">{{_('Guest')}}</span><br>
+                    <span id="location">
+                        {% if order.event.online %}
+                        {% if order.event.location_name %}
+                        <span>{{_('Event taking place online and at
```

Comment:
Add a full stop after %(location)s
`{{ _('Event taking place online and at %(location)s.', location="") }}`